### PR TITLE
Add "w" shorthand for width property

### PR DESCRIPTION
**Pull request by yodakaEngineer**
I personally implemented this because it was inconvenient for me to do without it.
If you feel strongly that implementing this is a bad idea, feel free to ignore this PR.
But since there is a short hand on the margin, this may just be a forgotten implementation.

### CODE REVIEW

File: packages/system/src/consumeStyleProps.ts [hunk 1]
```diff
@@ -42,8 +42,6 @@ export type StyledProps<T extends ThemeSystemType = ThemeSystemType> =
     MaskProps &
     ColumnProps<T> &
     BackgroundProps &
-    // eslint-disable-next-line @typescript-eslint/no-duplicate-type-constituents -- FIXME
-    EffectProps &
```
Comment (yodakaEngineer) [inline packages/system/src/consumeStyleProps.ts:46]: EffectProps was duplicate on line41. So remove it.

File: packages/system/src/props/layout.ts [hunk 2]
```diff
@@ -19,10 +24,16 @@ export type LayoutProps<T extends ThemeSystemType = ThemeSystemType> = Partial<
     CSSProperties<"float" | "clear"> &
     CSSProperties<"objectFit" | "objectPosition"> &
     CSSProperties<"resize"> &
-    CSSProperties<"verticalAlign">
+    CSSProperties<"verticalAlign"> & {
+      /**
+       * @see margin
```
Comment (poteboy) [inline packages/system/src/props/layout.ts:29]:
```suggestion
       * @see width
```

File: packages/system/src/consumeStyleProps.ts (hunk 1 above)
Comment (poteboy) [reply packages/system/src/consumeStyleProps.ts:46]: thx :)

**Review by poteboy — COMMENTED** [review]
Thanks for the PR!
Can you add tests to `consumeStyleProps.test.ts` to make sure that changes are valid?

**Review by poteboy — APPROVED** [review]
❤️

**Pull request merged**